File: ejemploJava/demo/src/main/java/com/example/OrdenamientoRapido.java
package com.example;

public class OrdenamientoRapido {
    // Ejemplo de Ordenamiento rápido en estructura de Datos
    public static void main(String[] args) {
        int[] numeros = { 9, 8, 7, 6, 5, 4, 3, 2, 1 };
        int[] numerosOrdenados = new int[9];
        int pivote = numeros[0];
        int contador = 0;

        // Ordenar números
        for (int i = 1; i < 9; i++) {
            if (numeros[i] < pivote) {
                numerosOrdenados[contador] = numeros[i];
                contador++;
            }
        }
        numerosOrdenados[contador] = pivote;
        contador++;
        for (int i = 1; i < 9; i++) {
            if (numeros[i] > pivote) {
                numerosOrdenados[contador] = numeros[i];
                contador++;
            }
        }

        // Mostrar números ordenados
        System.out.println("Números ordenados:");
        for (int i = 0; i < 9; i++) {
            System.out.print(numerosOrdenados[i] + " ");
        }
        System.out.println();
    }

}

File: ejemplosPython/ordenamiento_rapido.py
# Ejemplo de Ordenamiento rápido
numeros = [9, 8, 7, 6, 5, 4, 3, 2, 1]
numeros_ordenados = [0] * 9
pivote = numeros[0]
contador = 0

# Ordenar números
for i in range(1, 9):
    if numeros[i] < pivote:
        numeros_ordenados[contador] = numeros[i]
        contador += 1
numeros_ordenados[contador] = pivote
contador += 1
for i in range(1, 9):
    if numeros[i] > pivote:
        numeros_ordenados[contador] = numeros[i]
        contador += 1

# Mostrar números ordenados
print("Números ordenados:")
for i in range(9):
    print(numeros_ordenados[i], end=" ")
print()